requirements.txt

```diff
@@ -12,7 +12,7 @@ Jinja2==2.10
 pbr==5.1.1
 pexpect==4.6.0
 psutil==5.4.6; sys_platform!="win32" and sys_platform!="cygwin"
-PyYAML==3.13
+PyYAML>=5.1b3  # MIT
```

Comment:
We cannot depend on beta versions.

Reply:
Please add some comments on https://github.com/yaml/pyyaml/issues/259 to raise awareness of the needed release of pyyaml.

Reply:
Come to #pyyaml at Freenode to push this forward. Ingy is planning the release AFAIK but he is extremely cautious this time. I can't even push a testing PR hanging there since around the summer...
I can only say that everyone knows about this issue but what's needed is probably more collaboration and helping the maintainer. He even sometimes asks for changes feedback before creating actual PRs in the project he maintains, afraid to cause a shitload of useless negativity.
I wish I could dedicate some time to that project myself :(
Anyway, Ingy is quite responsive on IRC. I'm sure that if you offer him some help he'll be glad to accept it.